.gitignore

```diff
@@ -6,7 +6,7 @@ venv
 **/target/
 /pytest/output.log
 /pytest/venv/
-/pytest/*/__pycache__/
+/pytest/**/__pycache__/
```

Comment:
Nice. Ran the pytests today for the first time and got some unwanted `__pycache__` folders in my git worktree :sweat_smile:.

Reply:
I ran into the same issue with `**/target`, which [used to be](https://github.com/near/mpc/commit/79e4ac24ab7fb91056982710a69850c71ce60e78#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R6) `*/target`. I presume there may be more non-optimal stuff in `.gitignore`.